server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -1080,10 +1136,19 @@ private boolean hasNoRepeatedBlocks(ProgramDefinition programDefinition, long bl
     return programDefinition.getBlockDefinitionsForEnumerator(blockId).isEmpty();
   }
 
+  /**
+   * Returns if the currently displayed program is a draft or not. This is used to identify if the
```

Comment:
optional: IMO, that this returns if the program is a draft is a bit of an implementation detail. I think it's still fine to mention in the comment, but I would lead with the bit about this returning whether or not the view should be editable and then add that it's editable if it's a draft if you want to include more detail.

Reply:
Done